examples/albert/run_trainer.py

```diff
@@ -231,16 +199,20 @@ def main():
     set_seed(training_args.seed)
 
     config = AlbertConfig.from_pretrained(dataset_args.config_path, cache_dir=dataset_args.cache_dir)
-    tokenizer = AlbertTokenizerFast.from_pretrained(dataset_args.tokenizer_path, cache_dir=dataset_args.cache_dir)
+    try:
+        tokenizer = AlbertTokenizerFast.from_pretrained(dataset_args.tokenizer_path, cache_dir=dataset_args.cache_dir)
+    except OSError:
+        logger.fatal(f"No tokenizer data found in {dataset_args.tokenizer_path}, "
+                     f"please run ./tokenize_wikitext103.py before running this")
+        sys.exit(1)
+
```

Comment:
Before this change, it has shown `requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://huggingface.co/api/models/data/tokenizer` (a really obscure exception) In case of the absence of the tokenizer data.